pysat/_files.py

```diff
@@ -169,6 +177,26 @@ def __init__(self, sat, manual_org=False, directory_format=None,
                 # couldn't find stored info, load file list and then store
                 self.refresh()
 
+    def _filter_empty_files(self):
+        """Update the file list (files) with empty files ignored"""
+
+        keep_index = []
+        for i, fi in enumerate(self.files):
+            # create full path
+            fi_path = os.path.join(self.data_path, fi)
+            # ensure it exists
+            if os.path.exists(fi_path):
+                # check for size
+                if os.path.getsize(fi_path) > 0:
+                    # store if not empty
+                    keep_index.append(i)
+        # remove filenames as needed
+        if len(keep_index) < len(self.files.index):
+            # print('Found ' + str(len(self.files.index) - len(keep_index)) +
```

Comment:
Should we provide feedback about removing empty files? It could potentially be confusing for the user if pysat reports, say 100 files found, but the user can directly see 150 files in the relevant folder.

Reply:
A user message would be good.  Something along the line of "not counting empty files"

Reply:
I turned the feedback back on.